commown_devices/models/wizard_link_picking_po.py

```diff
@@ -96,6 +100,10 @@ def _compute_po_domain(self):
                 }
             }
 
+    @api.onchange("picking_id")
+    def _compute_po_id_is_readonly(self):
+        self.po_id_is_readonly = self.picking_id.purchase_id != False
```

Comment:
J'aurais juste mis bool(self.picking_id.purchase_id), notamment parce qu'on pourrait imaginer que ce soit un result set vide plutôt qu'une valeur "magique" comme False. Et en passant pour False le mot clef "is" est plus indiqué que == ou !=.

Reply:
Et en fait en y repensant tu ne t'en serais pas sorti avec un simple related="picking_id.purchase_id" ?

Reply:
Effectivement pour le result set vide ! Du coup j'ai modifié pour bool(self.picking_id.purchase_id), je préfère quand même ça au related car ça permet de garder en boolean, je trouve ça plus consistant pour un champ is_readonly